app/assets/javascripts/reducers/users.js

```diff
@@ -24,7 +24,7 @@ const SORT_DESCENDING = {
 export default function users(state = initialState, action) {
   switch (action.type) {
     case RECEIVE_USERS: {
-      const sortedUsers = sortByKey(action.data.course.users, state.sort.sortKey, 'username');
+      const sortedUsers = sortByKey(action.data.course.users, state.sort.sortKey, 'username', null, false, false, true);
```

Comment:
You forgot to remove the 'username', so there are too many params.

Reply:
Oh yeah. Removed it.

Reply:
Also thanks for the continous feedback you've provided on this. Learnt a lot!